app/views/applicants/new.html.erb

```diff
@@ -2,11 +2,19 @@
 
 <%= form_with url: "/applicants", method: :post, local: true do |form| %>
   <%= form.label :name %>
-  <%= form.text_field :name, null: false%>
+  <%= form.text_field :name, null: false%><br><br>
   <%= form.label :address %>
 
-  <%= form.text_field :address, null: false%>
+  <%= form.text_field :address, null: false%><br><br>
   <%= form.label :description %>
-  <%= form.text_field :description %>
+  <%= form.text_field :description %><br><br>
   <%= form.submit 'Create Application' %>
+
+
+  <% flash.each do |type, msg| %>
```

Comment:
this flash message block isn't needed here since it automatically renders on every view via the application.html.erb file :)